src/components/Workshop/WorkshopCard.scss

```diff
@@ -0,0 +1,28 @@
+@import '../../styles/Constants.scss';
+
+#workshop-background {
+    position:relative;
+    width: 331px;
+    height: 452px;
+    background-color: grey;
+    background-blend-mode: multiply;
+}
+
+#workshop-content {
+    position:absolute;
+    top:61%;
+    color:$neutral-color-95;
+    margin-left:20px;
+    width:50%;
+}
+
+#workshop-h4{
+    margin-block-start: 0em;
+    margin-block-end: 15px;
+}
+
+#workshop-h6{
+    margin-block-start: 0em;
+    margin-block-end: 15px;
+}
```

Comment:
Id o nazwie `workshop-h4/h6` nie ma prawa bytu, nazwij rzeczy po imineiu, jeden napis to label, drugi to tytuł eventu, nie lepiej w tym kierunku?

Poza tym, czemu korzystasz z tych marginów, nie lepiej bardziej powszechne rozwiązanie typu `margin-top`?

Zatrzymując się przy tych marginach, uważam że też się one nie nadadzą. Najwygodniej w całym komponencie byłoby skorzystać z flexboxa, który da nam tutaj jakiejkolwiek responsywności